rfcs/0011-longterm-storage.md

```diff
@@ -92,14 +93,40 @@ Transaction {
    instructions: vec![
        SystemInstruction::Move { tokens: 98 },
        LoaderInstruction::Write { offset: 0, bytes: [3,0,0,0,0,0,0,0,1,0,0,0,0,0,1,0,0,0,0,0,0,0,20,0,0,0,0,0,0,0,50,48,53,48,45,48,49,45,48,49,84,48,48,58,48,48,58,48,48,90,252,10,7,28,246,140,88,177,98,82,10,227,89,81,18,30,194,101,199,16,11,73,133,20,246,62,114,39,20,113,189,32,50,0,0,0,0,0,0,0,247,15,36,102,167,83,225,42,133,127,82,34,36,224,207,130,109,230,224,188,163,33,213,13,5,117,211,251,65,159,197,51,0,0,0,0,0,0]},
-       LoaderIntruction::Finalize,
-       SystemInstruction::Assign { program_id: [1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0] },
+       LoaderInstruction::Finalize,
    ],
    fee: 1,
    ...
 }
 ```
 
 After restoring the account, it should look identical to the original, minus the
-two transaction fees.
+three transaction fees.
+
+## Limitations
+
+### Clunky usage
+
+The method described above is an off-chain solution and therefore could be
+viewed as "clunky". Bringing the process on-chain would solve a number of
+problems:
+
+1. The API could be reduced to `Store(Pubkey)` and `Load(Pubkey)`
+2. The on-chain program could guarentee the restored version is identical
+   to the original
+3. One less transaction fee
+4. Could be utilized by any program to automatically reduce the cost of using it
+
+### Cannot restore owner
+
+It's possible the proposed solution only work for accounts owned by the
```

Comment:
Missing word? "the proposed solution would only work"

Reply:
Fixed, thanks!